options.go

```diff
@@ -392,6 +393,14 @@ func (o *Options) Validate() error {
 	msgs = validateCookieName(o, msgs)
 	msgs = setupLogger(o, msgs)
 
+	if o.RealClientIPHeader != "" {
+		if match, err := regexp.MatchString("^[a-zA-Z0-9-]+$", o.RealClientIPHeader); err != nil {
+			msgs = append(msgs, fmt.Sprintf("real_client_ip_header (%s) could not check regex: %s", o.RealClientIPHeader, err.Error()))
+		} else if !match {
+			msgs = append(msgs, fmt.Sprintf("real_client_ip_header (%s) not in valid format", o.RealClientIPHeader))
```

Comment:
Yes I think we can.  But I'm starting to wonder if we should break this down to a definite list of possible parameters.  Although `X-Forwarded-For`, `X-Real-IP`, and`X-ProxyUser-Ip` (maybe a few others) follow the same format, the `Forwarded` header definitely does not.  I'm not sure if I want to add support for the `Forwarded` header in this PR because A) it's complicated and B) I don't personally need it.

Reply:
Did it, of course.